### ISSUE TITLE
[Links] Add "link" mini-icon to other object-icons

### ISSUE THREAD

**Issue by charlesh88** [nasa/openmct#1588]
Add the "link" mini-icon object-icons such as the main view area header object and in Display Layout child frames. Request stems from conversation in email thread "WARP question" 5/18/17.

**Comment by larkin** [comment]
keep in mind that some people want to get rid of headers and frames in display layouts alltogether...  

(also can we defer this?)

**Comment by charlesh88** [comment]
@larkin Yep, as do I as a user-controllable option. Also, note that I didn't put this in a sprint...

**Comment by larkin** [comment]
> Also, note that I didn't put this in a sprint...
👍 💯 👍 ....🌮

**Comment by charlesh88** [comment]
Not sure what that all means, but I do like tacos.
